bigquery_etl/glam/templates/client_probe_counts_extract_v1.sql

```diff
@@ -0,0 +1,46 @@
+-- TODO: Remove deduping when dupes are fixed.
+WITH deduped AS (
```

Comment:
The de-duping was something we had problems with for desktop aggregations and which should be fixed now. Does this also have the issue with Glean on Fenix? If not, I'd remove this and see how the imports work. The imports use upserts which rely on there being a unique row per the dimensions.

Reply:
I haven't tried it without this. Do you know what the original reason for duplicates was?

Reply:
Two main reasons:
1. Some fields, e.g. os, had values of `NULL`, this interfered with our use of os = `NULL` to include all OSes and resulted in some rows appearing twice.
2. Buckets in the data for a given metric had different values so when grouping included buckets, some rows would appear more than once.

Both of these have since been resolved for desktop (except for the user counts query which does not exclude null OSes yet).

I suspect Glean data is more trustworthy and should not unexpectedly have incorrect/null values for fields but it's worth double checking before removing the deduping here.

Reply:
I removed the deduping and verified that the resulting aggregates violate the uniqueness constraint in the GLAM backend. I'll have to dig into this a bit more in the future.